pyproject.toml

```diff
@@ -29,7 +29,7 @@ packaging = "*"
 pandas = "^1.0"
 typing_extensions = ">=3.7.4.2"
 xmltodict = ">=0.12.0"
-xtiff = ">=0.6.2"
+xtiff = ">=0.6.3"
```

Comment:
This should be 0.6.4

Reply:
It is still 0.6.3 on PyPI: https://pypi.org/project/xtiff/